### PR TITLE
Add pullquote option to blockquote component

### PR DESCRIPTION
**Pull request by devnook**
Local preview at this url: http://localhost:8080/handbook/web-dev-components/#block-quotes

### CODE REVIEW

File: src/scss/next.scss [hunk 1]
```diff
@@ -179,15 +179,33 @@ dd + dt {
...
 blockquote {
+  padding-left: 24px;
+
+  cite {
+    font-size: get-size('size-0');
+  }
+}
+
+blockquote::before {
+  content: '”';
```
[the remark this answers is not in the document]
Comment (devnook) [reply src/scss/next.scss:190]: This dos not work if there is no paired close-quote present. Also, we want this to be "close-quote" and not open, which will not render if the open-quote is not used previously.

[the remark this answers is not in the document]
Comment (tomayac) [reply src/scss/next.scss:190]: Using either of the two seems to work in this example: https://ethereal-decorous-animal.glitch.me/.

[the remark this answers is not in the document]
Comment (devnook) [reply src/scss/next.scss:190]: Screenshot 2022-01-24 at 16 32 32

**Review by MichaelSolati — APPROVED** [review]
LGTM, just a two small tweaks in the handbook.

Also not sure about the `"` in the content, once that's resolved I'd say merge away.

File: src/scss/next.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomayac) [reply src/scss/next.scss:190]: Gotcha. #TIL, thank you!

File: src/site/content/en/handbook/web-dev-components/index.md [hunk 2]
```diff
@@ -224,14 +221,56 @@ at the end of a block quote:
...
+```html
+<blockquote data-type="pullquote">
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    Proin dictum a massa sit amet ullamcorper.
+  </p>
+  <cite>
+    Jon Doe
+  </cite>
+</blockquote data-type="pullquote">
```
[the remark this answers is not in the document]
Comment (devnook) [reply src/site/content/en/handbook/web-dev-components/index.md]: Oopsie:)

File: src/site/content/en/handbook/web-dev-components/index.md [hunk 3]
```diff
@@ -224,14 +221,56 @@ at the end of a block quote:
...
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    Proin dictum a massa sit amet ullamcorper.
+  </p>
+  <cite>
+    Jon Doe
+  </cite>
+</blockquote>
+
+To additionally emphasize a piece of text from the article, or a quote,
```
[the remark this answers is not in the document]
Comment (devnook) [reply src/site/content/en/handbook/web-dev-components/index.md]: Updated copy

**Pull request merged**